### PR TITLE
Navigation ui

### PR DESCRIPTION
**Pull request by ViachaslauPratasenia**

### CODE REVIEW

File: app/src/main/res/values-en/strings.xml [hunk 1]
```diff
@@ -0,0 +1,13 @@
+<?xml version="1.0" encoding="utf-8"?>
+ <resources>
+        <string name="app_name">Discovery</string>
```
Comment (Agent00krosh) [inline app/src/main/res/values-en/strings.xml]: rename to Discover

File: app/src/main/res/values-ru/strings.xml [hunk 2]
```diff
@@ -0,0 +1,11 @@
+<?xml version="1.0" encoding="utf-8"?>
+<resources>
+    <string name="app_name">Discovery</string>
```
Comment (Agent00krosh) [inline app/src/main/res/values-ru/strings.xml]: rename to Discover

File: app/src/main/res/values/strings.xml [hunk 3]
```diff
@@ -1,3 +1,11 @@
 <resources>
     <string name="app_name">Discovery</string>
```
Comment (Agent00krosh) [inline app/src/main/res/values/strings.xml]: rename to Discover

Comment (Scrollend723) [reply app/src/main/res/values/strings.xml]: you can make it untranslatable

**Pull request merged**